### PR TITLE
Fixes to master

### PR DESCRIPTION
**Pull request by lukechurch**
@devoncarew This switches to using the .package files and also brings in the kernel package which is now needed

### CODE REVIEW

File: lib/sdk_io.dart [hunk 1]
```diff
@@ -19,11 +19,12 @@ export 'sdk.dart';
...
  * This `DartSdk` implementation reads the SDK resources encoded into the
  * `compiler_unsupported` package as part of a build step. This does not rely
- * on being able to locate an SDK on disk. It _does_ need to have a populated
- * packages directory in the current working directory.
+ * on being able to locate an SDK on disk. It _does_ need to have a .packages
+ * file in the working directory
  */
 class DartSdkIO implements DartSdk {
-  static final _pkgPath = 'packages/compiler_unsupported/sdk';
+//  static final _pkgPath = 'packages/compiler_unsupported/sdk';
```
Comment (devoncarew) [inline lib/sdk_io.dart]: you can remove this commented out line

File: tool/updating.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 ## Updating `compiler_unsupported`
 
 - (optional) `rm -rf trunk`
+- rm -rf lib/src lib/_internal lib/sdk
 - update the `sdkTag` variable in tools/grind.dart
 - run `dart tool/grind.dart build`
 - run `dart tool/grind.dart validate`
+- run 'dart example/compiler.dart'
```
Comment (devoncarew) [inline tool/updating.md]: nit: this could use back-ticks

**Comment by devoncarew** [comment]
lgtm!

File: lib/sdk_io.dart (hunk 1 above)
Comment (lukechurch) [inline lib/sdk_io.dart]: done, thanks

File: tool/updating.md (hunk 2 above)
Comment (lukechurch) [inline tool/updating.md]: done, thanks

**Pull request merged**